src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -2776,21 +2776,11 @@ namespace WindowComplexManager {
                 if (state.dataGlobal->AnyLocalEnvironmentsInModel) {
                     if (state.dataSurface->SurfHasSurroundingSurfProperties(SurfNum)) {
                         SrdSurfsNum = state.dataSurface->SurfSurroundingSurfacesNum(SurfNum);
-                        if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
-                            state.dataSurface->Surface(SurfNum).ViewFactorSkyIR =
-                                state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor;
-                        }
-                        if (state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).SkyViewFactor != -1) {
-                            state.dataSurface->Surface(SurfNum).ViewFactorGroundIR =
-                                state.dataSurface->SurroundingSurfsProperty(SrdSurfsNum).GroundViewFactor;
-                        }
```

Comment:
This deleted code section is redundant because the view factors are already set at this stage. The view factors are constant; no need to reset them every time a window surface is simulated.

Reply:
:+1: